lib/search.js

```diff
@@ -34,11 +45,62 @@ function initialRequest (opts) {
 function extaractDeveloperId (link) {
   return link.split('?id=')[1];
 }
+// Mappings used for normal search results (no misspellings aka no suggested results)
```

Comment:
would it make this simpler if we dropped support for suggested/mispelling results? can you show how it looks on the ui and make a case for supporting this as part of the results here?
(there's a separate method for search suggestions not sure if they are the same sort of thing)